core/src/main/scala/org/apache/spark/sql/delta/files/TahoeCDCAddFileIndex.scala

```diff
@@ -16,58 +16,40 @@
 
 package org.apache.spark.sql.delta.files
 
-import org.apache.spark.sql.delta.{DeltaLog, Snapshot}
+import org.apache.hadoop.fs.Path
```

Comment:
org.apache.hadoop and org.apache.spark should be in separate import groups (with a blank line in between)
```
<java>

<scala>

<all other import groups>

<org.apache.spark.
```